taipy/core/_orchestrator/_orchestrator.py

```diff
@@ -328,3 +329,6 @@ def _check_and_execute_jobs_if_development_mode():
 
         if dispatcher := _OrchestratorFactory._dispatcher:
             dispatcher._execute_jobs_synchronously()
+
+    def is_dispatched(self, job_id: JobId) -> bool:
```

Comment:
I thought we want to move this to dispatcher?

Reply:
Yes, indeed. Let me change that.